milestone-delivery/deliveries/mobile-v3-swap-milestone-2.md

```diff
@@ -0,0 +1,19 @@
+- **Milestone Number:** 2
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which should be used for testing.
+
+| Number | Deliverable            | Specification                                                                                                                                                             | Link                                                                                                                     |
+| ------ | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| 0.     | Smart Contract Updates | The contracts were modified in order to be adapted to the existing Dart package capabilities and in order to better suit frontend needs                                   | https://github.com/SayNode/fondue-swap-vechain-contract Commit Hash: 513338c141c2667b7fc5cc430ae51020f0bf057f                     |
+| 1.     | Frontend: Design       | We will create the UI design in order to make it appealing and intuitive. Colors schemes, button formatting, location of elements, etc, will all be done i this phase.    | https://github.com/SayNode/fondue-swap-flutter Commit Hash: 8a98fad7e159828735fb2a09bc2f9c30032aa078                     |
+| 2a.    | Frontend: Development  | The previously planned UI will be build and organized.                                                                                                                    | https://github.com/SayNode/fondue-swap-flutter Commit Hash: 8a98fad7e159828735fb2a09bc2f9c30032aa078                     |
+| 2b.    | Frontend: Integration  | The frontend will connect to the contract                                                                                                                                 | https://github.com/SayNode/fondue-swap-flutter Commit Hash: 8a98fad7e159828735fb2a09bc2f9c30032aa078                     |
+| 2c.    | Frontend: Testing      | Several tests will be run to make sure everything works correctly: the UI works as intended and page load up speed/performance. The contract interaction is also checked. | https://github.com/SayNode/fondue-swap-flutter Commit Hash: 8a98fad7e159828735fb2a09bc2f9c30032aa078                     |
+| 2d.    | Frontend: Tutorial     | Simple guide on how to swap and set, check and change positions.                                                                                                          | https://github.com/SayNode/fondue-swap-flutter/blob/main/README.md Commit Hash: 8a98fad7e159828735fb2a09bc2f9c30032aa078 |

```

Comment:
Where can I find the application to try on my device?

Reply:
You can find the final .apk in this folder https://drive.google.com/drive/folders/1QXoDBafqEP0Hx5QLlLv6EGsjBmH-F3fq?usp=sharing